src/components/Image/index.native.js

```diff
@@ -18,9 +18,6 @@ function Image(props) {
     const {source, isAuthTokenRequired, session, ...rest} = props;
 
     let imageSource = source;
-    if (source && source.uri && typeof source.uri === 'number') {
-        imageSource = source.uri;
-    }
```

Comment:
@shubham1206agra My suggestion is we should keep this because this is our own wrapper component so here uri will support the number and reference not a problem. so we can keep clean all the place {uri: any} as @parasharrajat suggested.
https://github.com/Expensify/App/pull/26611/files#r1314509455